benchmark/infrastructure/BenchmarkMeasurementContainer.cpp

```diff
@@ -65,6 +66,24 @@ void to_json(nlohmann::json& j, const ResultGroup& resultGroup) {
                      {"metadata", resultGroup.metadata()}};
 }
 
+// ____________________________________________________________________________
+ResultTable::ResultTable(const std::string& descriptor,
+                         const std::vector<std::string>& rowNames,
+                         const std::vector<std::string>& columnNames)
+    : descriptor_{descriptor},
+      rowNames_{rowNames},
+      columnNames_{columnNames},
+      entries_(rowNames.size(), std::vector<EntryType>(columnNames.size())) {
+  // Having a table without any rows/columns makes no sense.
+  if (rowNames.size() == 0 || columnNames.size() == 0) {
```

Comment:
Should use `empty()` instead of `size() == 0`

Reply:
Done, see [ad6dc92](https://github.com/ad-freiburg/qlever/pull/946/commits/ad6dc927878e23baffa22fa22012664889273445).